TripCard/TripViewController.swift

```diff
@@ -0,0 +1,100 @@
+                         Trip(tripId: "Santorini001", city: "Santorini",
+                              country: "Greece", featuredImage: UIImage(named: "santorini"),
+                              price: 1800, totalDays: 7, isLiked: false),
+                         Trip(tripId: "NewYork001", city: "New York", country: "United States",
+                              featuredImage: UIImage(named: "newyork"),
+                              price: 900, totalDays: 3, isLiked: false),
+                         Trip(tripId: "Kyoto001", city: "Kyoto", country: "Japan",
+                              featuredImage: UIImage(named: "kyoto"),
+                              price: 1000, totalDays: 5, isLiked: false)
+    ]
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        backgroundImageView.image = UIImage(named: "cloud")
+        let blurEffect = UIBlurEffect(style: .dark)
+        let blurEffectView = UIVisualEffectView(effect: blurEffect)
+        blurEffectView.frame = view.bounds
+        backgroundImageView.addSubview(blurEffectView)
+        
+        collectionView.backgroundColor = UIColor.clear
+        
+        if UIScreen.main.bounds.size.height == 568.0 {
+            if let flowLayout = self.collectionView.collectionViewLayout as? UICollectionViewFlowLayout {
+                flowLayout.itemSize = CGSize(width: 250.0, height: 330.0)
+            }
+        }
+    }
+
+    override var preferredStatusBarStyle: UIStatusBarStyle {
+        return .lightContent
+    }
+}
+
+extension TripViewController: UICollectionViewDataSource {
+    
+    func collectionView(_ collectionView: UICollectionView, numberOfItemsInSection section: Int) -> Int {
+        return trips.count
+    }
+    
+    func collectionView(_ collectionView: UICollectionView, cellForItemAt indexPath: IndexPath) -> UICollectionViewCell {
+        let cell = collectionView.dequeueReusableCell(withReuseIdentifier: "Cell", for: indexPath) as? TripCollectionViewCell
+        
+        guard let safeCell = cell else {
+            return UICollectionViewCell()
+        }
+        
+        safeCell.delegate = self
+        safeCell.cityLabel.text = trips[indexPath.row].city
+        safeCell.countryLabel.text = trips[indexPath.row].country
+        safeCell.imageView.image = trips[indexPath.row].featuredImage
+        safeCell.priceLabel.text = "$\(String(trips[indexPath.row].price))"
+        safeCell.totalDaysLabel.text = "\(trips[indexPath.row].totalDays) days"
+        safeCell.isLiked = trips[indexPath.row].isLiked
+     
+        safeCell.layer.cornerRadius = 4.0
+        
+        return safeCell
+    }
+}
+
+extension TripViewController: TripCollectionViewCellDelegate {
+    
+    func didLikeButtonPressed(cell: TripCollectionViewCell) {
+        if let indexPath = collectionView.indexPath(for: cell) {
+            trips[indexPath.row].isLiked = trips[indexPath.row].isLiked ? false : true
```

Comment:
i think it is easier to read a toggle action like this (instead of ternary operator):
trips[indexPath.row].isLiked = !trips[indexPath.row].isLiked